silkrpc/core/evm_trace.cpp

```diff
@@ -1133,7 +1165,7 @@ asio::awaitable<TraceCallResult> TraceCallExecutor<WorldState, VM>::execute(std:
     tracers.push_back(tracer);
 
     EVMExecutor<WorldState, VM> executor{io_context_, database_reader_, *chain_config_ptr, workers_, block_number};
-    for (auto idx = 0; idx < index; idx++) {
+    for (auto idx = 0; idx < transaction.transaction_index; idx++) {
```

Comment:
This `execute` method overload is used also by other APIs, so just to double-check: is it correct to use `transaction.transaction_index` instead of `index` here also for other APIs?

Reply:
I did all checks as I know this method is used by several API. 
In addition it is better to use the `transaction_index `field to give `index` param additional meaning (ie, the one added by trace_callMany)